inc/helpers/namespace.php

```diff
@@ -276,7 +276,7 @@ function get_source_book( $book_url, $checked = [] ) {
  *
  * @param string $book_url The URL of the book to trace.
  *
- * @return array The metadata array.
+ * @return string The metadata array.
```

Comment:
The `@return` type is correct but the description isn't:

```suggestion
 * @return string The URL of the book's original source.
```